web/sites/default/settings.php

```diff
@@ -2,264 +2,80 @@
-//  - "*.gov.au" domains (PROD)
-//  - "*.org.au" domains (PROD)
-//
-// When the domain likely does not have valid SSL, then HSTS is disabled
-// explicitly (to prevent the database values being used).
-//
-// @see https://govdex.gov.au/jira/browse/GOVCMS-1109
-// @see http://cgit.drupalcode.org/seckit/tree/includes/seckit.form.inc#n397
-//
-if (preg_match("~^.+(\.gov\.au|\.org\.au)$~i", $_SERVER['HTTP_HOST'])) {
-  $config['seckit.settings']['seckit_ssl']['hsts'] = TRUE;
-  $config['seckit.settings']['seckit_ssl']['hsts_max_age'] = 31536000;
-  $config['seckit.settings']['seckit_ssl']['hsts_subdomains'] = FALSE;
-}
-else {
-  $config['seckit.settings']['seckit_ssl']['hsts'] = FALSE;
-  $config['seckit.settings']['seckit_ssl']['hsts_max_age'] = 0;
-  $config['seckit.settings']['seckit_ssl']['hsts_subdomains'] = FALSE;
-}
-
-// Lagoon Database connection
-if (getenv('LAGOON')) {
-  $databases['default']['default'] = [
-    'driver' => 'mysql',
-    'database' => getenv('MARIADB_DATABASE') ?: 'drupal',
-    'username' => getenv('MARIADB_USERNAME') ?: 'drupal',
-    'password' => getenv('MARIADB_PASSWORD') ?: 'drupal',
-    'host' => getenv('MARIADB_HOST') ?: 'mariadb',
-    'port' => 3306,
-    'charset' => 'utf8mb4',
-    'collation' => 'utf8mb4_general_ci',
-  ];
-}
-
-// Lagoon Solr connection
-if (getenv('LAGOON')) {
-  $config['search_api.server']['backend_config']['connector_config']['host'] = getenv('SOLR_HOST') ?: 'solr';
-  $config['search_api.server']['backend_config']['connector_config']['path'] = '/solr/' . getenv('SOLR_CORE') ?: 'drupal';
-}
-
-// Lagoon Varnish & reverse proxy settings
-if (getenv('LAGOON')) {
-  $varnish_control_port = getenv('VARNISH_CONTROL_PORT') ?: '6082';
-  $varnish_hosts = explode(',', getenv('VARNISH_HOSTS') ?: 'varnish');
-  array_walk($varnish_hosts, function (&$value, $key) use ($varnish_control_port) {
-    $value .= ":$varnish_control_port";
-  });
-
-  $settings['reverse_proxy'] = TRUE;
-  $settings['reverse_proxy_addresses'] = array_merge(explode(',', getenv('VARNISH_HOSTS')), ['varnish']);
-  $settings['varnish_control_terminal'] = implode($varnish_hosts, " ");
-  $settings['varnish_control_key'] = getenv('VARNISH_SECRET') ?: 'lagoon_default_secret';
-  $settings['varnish_version'] = 4;
-}
-
-// Redis configuration.
-if (getenv('LAGOON') && (getenv('ENABLE_REDIS'))) {
-  $settings['redis.connection']['interface'] = 'PhpRedis';
-  $settings['redis.connection']['host'] = getenv('REDIS_HOST') ?: 'redis';
-  $settings['redis.connection']['port'] = 6379;
-
-  $settings['cache_prefix']['default'] = getenv('LAGOON_PROJECT') . '_' . getenv('LAGOON_GIT_SAFE_BRANCH');
+// Set by Drupal core.
+/* @var $app_root string */
 
-  # Do not set the cache during installations of Drupal
-  if (!drupal_installation_attempted()) {
-    $settings['cache']['default'] = 'cache.backend.redis';
+// Path to standardized platform-wide settings includes.
+/* @var $govcms_includes string */
```

Comment:
misplaced comment. should be above line https://github.com/govCMS/govcms8-scaffold-paas/pull/12/files#diff-2c8464f955da8bf797f47fc817ce58b1R48